heap_sort.py

```diff
@@ -0,0 +1,21 @@
+from heaps.min_heap import MinHeap
+from heaps.min_heap import HeapNode
+
+def heap_sort(list):
```

Comment:
⚠️ This is incomplete as you're not removing things from the heap until the heap is empty and the list is sorted.